tests/test_jsonyamlxml_writer.py

```diff
@@ -1,36 +1,74 @@
+import glob
 import os
+from datetime import datetime
 
 import pytest
 
 from spdx.document import Document
-from spdx.package import Package, ExternalPackageRef
+from spdx.package import Package, ExternalPackageRef, PackagePurpose
 from spdx.parsers.parse_anything import parse_file
-from spdx.writers import json
+from spdx.writers import json, write_anything
 from tests.test_rdf_writer import minimal_document
 
 
 @pytest.fixture
 def temporary_file_path() -> str:
-    temporary_file_path = "temp_external_references.json"
+    temporary_file_path = "temp_external_references"
     yield temporary_file_path
-    os.remove(temporary_file_path)
+    file_with_ending = glob.glob(temporary_file_path + "*")
+    for file in file_with_ending:
+        os.remove(file)
 
 
-def test_external_package_references(temporary_file_path) -> None:
+@pytest.mark.parametrize("out_format", ['yaml', 'xml', 'json'])
+def test_external_package_references(temporary_file_path: str, out_format: str) -> None:
```

Comment:
Let's remove the type annotations on tests

Reply:
removed that commit

Reply:
The type hint is still there. But this is really minor....